runtime/pubsub/manager_internal.go

```diff
@@ -136,7 +136,7 @@ func registerProvider(p func(mgr *Manager) provider) {
 
 var _ types.PushEndpointRegistry = (*Manager)(nil)
 
-func (mgr *Manager) RegisterPushSubscriptionHandler(id types.SubscriptionID, handler types.PushEndpointHandler) {
+func (mgr *Manager) RegisterPushSubscriptionHandler(id types.SubscriptionID, handler http.HandlerFunc) {
```

Comment:
I needed access to the raw `ResponseWriter` to do the SSE, which meant dropping the nice `func(req) error` we had before :(